src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -593,7 +593,8 @@ void MavlinkReceiver::handle_message_command_both(mavlink_message_t *msg, const
 		result = handle_request_message_command(MAVLINK_MSG_ID_STORAGE_INFORMATION);
 
 	} else if (cmd_mavlink.command == MAV_CMD_SET_MESSAGE_INTERVAL) {
-		if (set_message_interval((int)roundf(cmd_mavlink.param1), cmd_mavlink.param2, cmd_mavlink.param3)) {
+		if (set_message_interval((int)roundf(cmd_mavlink.param1), cmd_mavlink.param2, 0, cmd_mavlink.param3, cmd_mavlink.param4,
```

Comment:
Note, data_rate now set to 0 always, not `cmd_mavlink.param3`

Reply:
Note, below I wanted to do `cmd_mavlink.param5`, but the value is templated, and that won't exist if sent in an int. So I use the vehicle_command values.